tutorials/preparing-the-vm.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: page
+title: Preparing the Virtual Machine
+---
+
+Complete the following steps to run Kite example code on a Cloudera Quickstart VM.
+
+* Install a VirtualBox or VMWare [Cloudera QuickStart VM][getvm] version 5.2 or later.
+* In that VM, run the following command from a terminal window. This command clones a local copy of Kite code examples you can build and run.
+
+```bash
+git clone https://github.com/kite-sdk/kite-examples.git
+```
+
+* If you haven't already done so, download the [`kite-dataset` CLI JAR][install-cli].
```

Comment:
It would be better to use "command" to describe the CLI rather than referring to it as the"CLI JAR" or "executable JAR". The format doesn't matter much, what matters is that it is a command you can run directly. That's what you mean in nearly every case.